src/server.h

```diff
@@ -1469,6 +1469,10 @@ struct malloc_stats {
     size_t allocator_resident;
     size_t allocator_muzzy;
     size_t allocator_frag_smallbins_bytes;
+    size_t lua_allocator_allocated;
+    size_t lua_allocator_active;
+    size_t lua_allocator_resident;
+    size_t lua_allocator_frag_smallbins_bytes;
```

Comment:
maybe we need to show these in MEMORY STATS?
or maybe in INFO next to `used_memory_lua`?
i'm not sure.

Reply:
i think it's need, yesterday i found that actually the gap between `used_memory_lua` and `lua_allocator_allocated` is still big, but as the lua consume more memory the gap gets smaller.